[IMP] Administration: Add client database info

pabr-odoo opened the pull request:
Task: https://www.odoo.com/odoo/project/3835/tasks/4568894

Adding a new section explaining how to add a client database to an Odoo.com account, while also overhauling the introduction of the doc.

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/12256.png)](https://mergebot.odoo.com/odoo/documentation/pull/12256)

pabr-odoo commented:
Hey @dikd-odoo this PR is ready for peer review whenever you can get to it. Thank you!

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+reset an Odoo.com account password, and how to add two-factor authentication.
+
+Add a client database to an Odoo.com account
+==============================================
```
```suggestion
============================================
```

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
-This documentation is dedicated to edits made to an Odoo.com account. The following processes
-describe how to delete an Odoo.com account, and how to change the password on an Odoo.com account.
+This document describes how to edit and manage an Odoo.com account. This includes common processes,
+such as adding a client database to an Odoo.com account, how to delete an Odoo.com account, how to
```
```suggestion
such as how to add a client database to an Odoo.com account, how to delete an Odoo.com account, how to
```

dikd-odoo replied:
Edited for parallel listing syntax

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
 
-This documentation is dedicated to edits made to an Odoo.com account. The following processes
-describe how to delete an Odoo.com account, and how to change the password on an Odoo.com account.
+This document describes how to edit and manage an Odoo.com account. This includes common processes,
```
```suggestion
This document describes how to edit and manage an Odoo.com account. This includes common processes
```

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
-describe how to delete an Odoo.com account, and how to change the password on an Odoo.com account.
+This document describes how to edit and manage an Odoo.com account. This includes common processes,
+such as adding a client database to an Odoo.com account, how to delete an Odoo.com account, how to
+reset an Odoo.com account password, and how to add two-factor authentication.
```
```suggestion
reset an account password, and how to add two-factor authentication.
```

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+desired Odoo.com account needs to be added to the database as a user.
+
+First, sign in to the Odoo.com account that created the desired database. In the main Odoo dashboard
+of the database, navigate to :menuselection:`**Settings** --> Manage Users`. Click the
```
```suggestion
of the database, navigate to :menuselection:`Settings --> Manage Users`. Click the
```

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+   :alt: The `Users Invitations` pop-up window with the options to `Confirm` or `Cancel`.
+
+Click :guilabel:`Confirm`, and the :guilabel:`Users` list appears. An invitation is automatically
+sent to the email address. Once confirmed, the database now appears in the **My Databases** page of
```
```suggestion
sent to the email address. Once confirmed, the desired database now appears in the **My Databases** page of
```

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+
+Click :guilabel:`Confirm`, and the :guilabel:`Users` list appears. An invitation is automatically
+sent to the email address. Once confirmed, the database now appears in the **My Databases** page of
+the account.
```
```suggestion
the target Odoo.com account.
```

dikd-odoo replied:
Optional since it could be considered redundant, but I would consider being very explicit about which Odoo.com account the database appears in after the steps

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+sent to the email address. Once confirmed, the database now appears in the **My Databases** page of
+the account.
+
+.. tip::
```
Optional suggestion: could be useful to add a screenshot of the recommended settings

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+   :guilabel:`Settings`.
+
+.. example::
+   Deco Addict wishes to have the database 'Stealthy Wood' appear in their 'My Databases'
```
```suggestion
   Deco Addict wishes to have the database **Stealthy Wood** appear in their **My Databases**
```

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+
+.. example::
+   Deco Addict wishes to have the database 'Stealthy Wood' appear in their 'My Databases'
+   page, but 'Stealthy Wood' was created by Mitchell Admin. To add the account, a user with
```
```suggestion
   page, but **Stealthy Wood** was created by a different user, Mitchell Admin. To add the account, a user with
```

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+.. example::
+   Deco Addict wishes to have the database 'Stealthy Wood' appear in their 'My Databases'
+   page, but 'Stealthy Wood' was created by Mitchell Admin. To add the account, a user with
+   administrator access of 'Stealthy Wood' adds Deco Addict as a user of 'Stealthy Wood'.
```
```suggestion
   administrator access of the **Stealthy Wood** database adds Deco Addict as a user of **Stealthy Wood**.
```

dikd-odoo commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,43 @@
+   Deco Addict wishes to have the database 'Stealthy Wood' appear in their 'My Databases'
+   page, but 'Stealthy Wood' was created by Mitchell Admin. To add the account, a user with
+   administrator access of 'Stealthy Wood' adds Deco Addict as a user of 'Stealthy Wood'.
+   Now when Deco Addict visits their 'My Databases' page, 'Stealthy Wood' appears.
```
```suggestion
   Now when Deco Addict visits their **My Databases** page, the **Stealthy Wood** database appears, and they have user access.
```

dikd-odoo reviewed (approved):
Hi @pabr-odoo great job on this PR! The breakdown of the steps was super clear and the flow worked perfectly. I just left a handful of formatting and readability suggestions; once you've taken a look at those, this is ready for final review. Thanks!

pabr-odoo commented:
Hey @Felicious this PR is ready for final review whenever you can get to it. Thank you!

Felicious commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,44 @@
+such as how to add a client database to an Odoo.com account, how to delete an Odoo.com account, how
+to reset an account password, and how to add two-factor authentication.
+
+Add a client database to an Odoo.com account
```
After reading through the section, I think the title might not fully match the instructions provided here. From the title, it seems like we're connecting two databases, but it looks like the instructions are actually about inviting a user to another database by email.

How about we go with something like: 'Add user to Odoo.com' or 'Grant users access to database' instead?

Felicious reviewed (changes requested):
Hi @pabr-odoo 

I had some confusion about this PR, and I’d like to clarify a couple of points.

First, regarding the point valuation: As outlined in our [Point system](https://docs.google.com/document/d/13ehNrKNdIna92vodzWgx5HEzIfj5SpymIT9WfotT0Yc/edit?tab=t.0#heading=h.a0402hfhkggd) team guidelines, a PR is generally assigned 3 points when it's a 50-80% change. In this case, 38 lines were changed in an 80-line document, which seems to fall within that 50-80% range. Could we revisit the points to ensure they align with the guidelines?

Additionally, the instructions provided seemed a bit more detailed than expected for the task, which was simply about adding users to a database. While it’s important to be thorough, the level of detail could be streamlined to ensure clarity and ease of use for the reader.

Thanks for your work on this, and I’m happy to discuss further if needed.

pabr-odoo commented:
Hi @Felicious thank you for the thorough review! I adjusted the point value here and in the task, and implemented your suggestions. Please give it another look when you get the chance. Thanks!

Felicious commented on content/administration/odoo_accounts.rst:
```diff
@@ -2,8 +2,33 @@
+==============================
+
+A database can be linked to an Odoo.com account that was created by a different Odoo.com account.
+This allows the database to appear in the **My Databases** page of the account. To do so, the
```
We use italics when referencing a page we're not directly looking at

Felicious reviewed (approved):
LGTM @pabr-odoo (:

pabr-odoo commented:
Hey @samueljlieber this PR is ready for tech review when you can get to it. Thanks!

samueljlieber commented:
@robodoo r+
